### ISSUE TITLE
Change user@host.topdomain to point to the F#Un user instead of an email

### ISSUE THREAD

**Issue by brolund** [agical/friendsunhosted#22]
Links email addresses in text are set with a ... href="mailto:user@host.topdomain" ... , but the default should of course be to a page displaying the info about a F#Un user, and a mouse over should probably give some of that info. 

Instead to get an mailto link, mailto:user@hds.top should be used, just as http: or the like is used for addresses.

**Comment by jancborchardt** [comment]
Yeah, good point! Instead of the mailto: syntax, how about automatically checking if the address has a remoteStorage attached to it – in which case it would link to a F#Un profile. If not, it would be a mailto.

**Comment by brolund** [comment]
Neat idea. What happens if you host your store on your laptop and close the lid? ;-)

I think we need to accept that stores will come and go since that is a part of the unhosted, distributed, networky, paradigm. I would like the application to behave consistently in the face of uncertainty, so to speak.

**Comment by johlrogge** [comment]
+1

**Comment by jancborchardt** [comment]
Yeah, good point. Let’s just assume every address posted there is a remoteStorage. If it’s not, it can just have an empty profile page saying »this user does not seem to have a F#Un account« and then something like »send them an email instead« (also including some piece about how they should join).
